plugins/modules/api.py

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: api
+
+author:
```

Comment:
This is already being covered in #198. I'm confused. Could you please keep the changes in separate PRs? it is very difficult to catch up.

Reply:
The problem is that this branch was branched from https://github.com/ansible-collections/servicenow.itsm/pull/198. Could https://github.com/ansible-collections/servicenow.itsm/pull/198 please be merged first so I can rebase this branch and changes that don't belong here won't be visible?